lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -374,6 +376,13 @@ class IdentityCreatorController extends BaseController {
     final sortOrder = isDefaultIdentitySupported.isTrue
       ? UnsignedInt(isDefaultIdentity.value ? 0 : 100)
       : null;
+
+    final publicAssetsInIdentityArguments = PublicAssetsInIdentityArguments(
+      htmlSignature: signatureHtmlText ?? '',
+      identityActionType: actionType.value,
```

Comment:
why we need to keep `identityActionType` inside `PublicAssetsInIdentityArguments`. How about to move it to `CreateNewIdentityRequest` or `EditIdentityRequest` or abstract class of these?

Reply:
Done